lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -102,8 +102,8 @@ def run
 
           latest_time_series = nil
           client.each_report_row do |row|
-            time = row[task["time_series"]]
-            next if last_record_time && time <= last_record_time
+            time = row["ga:dateHour"] ? row["ga:dateHour"] : row["ga:date"]
+            next if last_record_time && time.utc <= last_record_time.utc
```

Comment:
Well, isn't `row[task["time_series"]]` the original code already switching ga:dateHour and ga:date?
If so, could you give me the example config or something like so?